windows.md

```diff
@@ -397,7 +397,59 @@ With those compatibility things out of the way, you're ready to start the system
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
+
+    If opening Docker Desktop or running the command above shows a message that says `Docker Desktop - Windows HyperVisor is not Present`, you may need to enable virtualization on your machine.
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    First check if virtualization is enabled on your computer: To do this, open PowerShell as an administrator and run the following command:
+    ```bash
+    systeminfo | findstr /C:"Hyper-V" /C:"Virtualization"
```

Comment:
- do we need the extra `Hyper-V` check here?
- do we need the `/C:`? can we just do `systeminfo | findstr Virtualization`?

Reply:
We don't need the `/C:`
I did the command like this `systeminfo | findstr "Hyper-V Virtualization"` instead so that we can search for the two instances.  I expect that different Windows computers will call this functionality either `Hyper-V` or `Virtualization`, and I would want to search for both. However, 

in my Windows 10 and 11 Pro with Lukas's 10 Home, It is all called `Hyper-V` as searching for `Virtualization` did not produce any output

Maybe we should just have this `systeminfo | findstr Hyper-V`

62664824c1626b2307eb9fa836b38f582bcdb57a